### PR TITLE
[Web Share article] Add new third-party requirements

### PR DESCRIPTION
**Pull request by tomayac**
Changes proposed in this pull request:

- Add new third-party requirements

### CODE REVIEW

File: src/site/content/en/blog/web-share/index.md [hunk 1]
```diff
@@ -118,6 +119,21 @@ Certain types of audio, image, pdf, video, and text files can be shared.
 See [Permitted File Extensions in Chromium](https://docs.google.com/document/d/1tKPkHA5nnJtmh2TgqWmGSREUzXgMUFDL6yMdVZHqUsg/edit?usp=sharing)
 for a complete list. More file types may be added in the future.
 
+## Sharing in third-party iframes
+
+If you want to trigger the share action from within a third-party iframe,
```
Comment (rachelandrew) [inline src/site/content/en/blog/web-share/index.md:124]:
```suggestion
To trigger the share action from within a third-party iframe,
```

File: src/site/content/en/blog/web-share/index.md [hunk 2]
```diff
@@ -118,6 +119,21 @@ Certain types of audio, image, pdf, video, and text files can be shared.
...
+you need to embed the iframe with the `allow` attribute with a value of `web-share`:
```
Comment (rachelandrew) [inline src/site/content/en/blog/web-share/index.md:125]:
```suggestion
embed the iframe with the `allow` attribute with a value of `web-share`:
```

**Review by rachelandrew — APPROVED** [review]
see small edit.

**Comment by marcoscaceres** [comment]
This is great! Thanks @tomayac 🙏

**Pull request merged**